docs/apps/01-parity-signer.md

```diff
@@ -0,0 +1,32 @@
+---
+id: 01-parity-signer
+title: KILT & Parity Signer
+---
+
+For the more advanced users, it is possible to configure [Parity Signer](https://www.parity.io/technologies/signer/) to work as a cold or hot wallet with KILT and PolkadotJS Apps. Please follow the installation instructions for iOS and Android from the official website.
+
+## Configure Parity Signer to support KILT
+
+To add KILT Spiritnet to Parity Signer, visit PolkadotJS Apps and go to **KILT Spiritnet > Settings > Metadata** (link [here](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io%2F#/settings/metadata)) and scan the QRCode. This will result in a new network, KILT Spiritnet, being added to the list of networks for which Parity Signer can create accounts and sign transactions.
```

Comment:
W ... be nice to have new line after each sentence. Since GIT is line based it's better readable in new PRs.

Reply:
W ... also be cool to have something meaningful in the link. Like you did with parity signer. The link is highlighted and if *here* is highlighted that doesn't give any information.

```suggestion
To add KILT Spiritnet to Parity Signer, visit [PolkadotJS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io%2F#/settings/metadata) and go to **KILT Spiritnet > Settings > Metadata** and scan the QRCode. This will result in a new network, KILT Spiritnet, being added to the list of networks for which Parity Signer can create accounts and sign transactions.
```